WIP since this fails on Travis, with the following error:
```
❌  ld: unknown option: -platform_version
157
158
159
160❌  clang: error: linker command failed with exit code 1 (use -v to see invocation)
161
162
163** BUILD FAILED **
164
165
166The following build commands failed:
167	Ld /Users/travis/build/livecode/livecode/_build/mac/Debug/perfect-target normal x86_64
168(1 failure)
169make[1]: *** [compile-mac] Error 65
170make: *** [all-mac] Error 2
```

--- a/config/mac.gypi
+++ b/config/mac.gypi
@@ -30,7 +30,7 @@
 		'STRIP_INSTALLED_PRODUCT': 'NO',
 		'CLANG_LINK_OBJC_RUNTIME': 'NO',
 		'CLANG_CXX_LANGUAGE_STANDARD': 'c++11',
-        'CLANG_CXX_LIBRARY': 'libc++'
+		'CLANG_CXX_LIBRARY': 'libc++'
 	},
 	
 	'target_defaults':
@@ -47,6 +47,7 @@
 			'debug_info_suffix': '.dSYM',
 			
 			'silence_warnings': 0,
+			'travis': '<!(echo ${TRAVIS})',
 		},
 		
 		'target_conditions':
@@ -68,6 +69,21 @@
 						'_SERVER',
 						'_MAC_SERVER',
 					],
+				},
+			],
+			[
+				'travis == ""',
+				{
+					'xcode_settings':
+					{
+						'OTHER_LD_FLAGS':
+						[
+							'-Wl,-platform_version',
+							'-Wl,macos',
+							'-Wl,10.9',
+							'-Wl,10.9',
+						],
+					},
 				},
 			],
 			[

--- a/docs/notes/bugfix-23176.md
+++ b/docs/notes/bugfix-23176.md
@@ -1,0 +1,1 @@
+# Ensure externals are not blocked by hardened runtime

